index.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet">
+</head>
+<body>
+   <header>
+       <nav class="navbar">
+           <div class="toolbar">
+               <img src="./assets/images/bars/Time.svg" alt="toolbar">
+               <img src="./assets/images/bars/Group4.svg" alt="toolbar">
+           </div>
+           <div class="menubar">
+            <p>North Code</p>
+            <img src="./assets/images/bars/IconMenu.png" alt="menuicon">
+           </div>
+       </nav>
+   </header>
+
+<section class="headline">
+  <div class="headtext">
+      <h1 class="primary-text"> Kinginthenorth codez</h1>
+      <h3 class="sub-text"> Software Developer</h3>
+  </div>
+  <div class="surport-text">
+      <div class="intro">
+        <p>
+            I can help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded,  
+            don’t hestiate to contact me.
+          </p>
+      </div>
+      <img src="./assets/images/Devider.svg" alt="line">
```

Comment:
- [ ] I'll suggest you kindly use the `<hr>` tag for the divider instead of image